### ISSUE TITLE
Comments should not be parsed when inlining SVG

### ISSUE THREAD

**Issue by IlCallo** [oliverfindl/vue-svg-inline-loader#33]
I'm trying to put an example on how to use this loader into one of my Vue files comment, but build fails because inlining happens even inside comments (even inside `script` and `style` tags, actually).

Is there a way to limit inlining to `template` tag and avoid it when inside comments?
Eg these will both break build process.
- `// Ex. <img svg-inline src="../path/to/icon.svg" alt="Title"/>`
- `<!-- Ex. <img svg-inline src="../path/to/icon.svg" alt="Title"/> -->`
- `/* Ex. <img svg-inline src="../path/to/icon.svg" alt="Title"/> */`

**Comment by oliverfindl** [comment]
Hello,

this is true. This loader replaces all occurrences in code. I can fix that fail while building (I guess it's because it's inserting multi-line SVG into your line comment) by removing unnecessary whitespace from SVG and therefore making it one-liner, but it will still transform code in comments. I'm not going to write logic to parse out all comments from file (or just from `template` tag) and replace them back before returning it to Webpack. Not to mention, it would be hard to parse line comments (starting with `//`). This is kinda out of scope of this loader.

Better solution would be to implement proper HTML parser (e.g.: [htmlparser2](https://github.com/fb55/htmlparser2)), but this will require complete rewrite of loader. Except that there are multiple problems to solve I'm aware of. E.g.: 
- returning back (virtual) DOM / HTML as string
- Vue syntax is not valid HTML syntax

If you have other solution in mind, feel free to suggest it here.

Thanks

**Comment by IlCallo** [comment]
Staying into RegEx world, maybe adding escape delimiters?
Something like "ignore replacement if the code is between two backslashes/backtick/hashes/some other unluckily used character"?

It's an hack, but it solves the problem easily and I think it can be checked with minor tweaks to the RegEx.
Or just a little disclaimer on the README which documents this behaviour and warns about it.

**Comment by oliverfindl** [comment]
Hello,

that hack, you suggested, it's actually pretty easy to implement, because I got there functionality to match wrapping single and double quotes around image tag. So it's possible to extend it. In fact, I already did it.

I updated loader and published it to [npm](https://www.npmjs.com/package/vue-svg-inline-loader). Current version is [1.4.3](https://github.com/oliverfindl/vue-svg-inline-loader/releases/tag/v1.4.3).

You can wrap it in hash (`#`) or triple backtick (` ``` `) characters. Single backtick it's not possible to use, because it can be used in script tag (for template literals) as well.

Please give me feedback if it works as expected.

Thanks

**Comment by IlCallo** [comment]
Perfect, thank you!
